ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/application-traffic-recognition.md

```diff
@@ -0,0 +1,176 @@
+# application-traffic-recognition
+
+## Table of Contents
+
+- [Management](#management)
+  - [Management Interfaces](#management-interfaces)
+- [Application Traffic Recognition](#application-traffic-recognition)
+  - [Applications](#applications)
+  - [Application Profiles](#application-profiles)
+  - [Categories](#categories)
+  - [Field Sets](#field-sets)
+  - [Router Application-Traffic-Recognition Device Configuration](#router-application-traffic-recognition-device-configuration)
+
+## Management
+
+### Management Interfaces
+
+#### Management Interfaces Summary
+
+##### IPv4
+
+| Management Interface | description | Type | VRF | IP Address | Gateway |
+| -------------------- | ----------- | ---- | --- | ---------- | ------- |
+| Management1 | oob_management | oob | MGMT | 10.73.255.122/24 | 10.73.255.2 |
+
+##### IPv6
+
+| Management Interface | description | Type | VRF | IPv6 Address | IPv6 Gateway |
+| -------------------- | ----------- | ---- | --- | ------------ | ------------ |
+| Management1 | oob_management | oob | MGMT | - | - |
+
+#### Management Interfaces Device Configuration
+
+```eos
+!
+interface Management1
+   description oob_management
+   vrf MGMT
+   ip address 10.73.255.122/24
+```
+
+## Application Traffic Recognition
+
+### Applications
+
+#### IPv4 Applications
+
+| Name | Source Prefix | Destination Prefix | Protocol | Protocol Ranges | Tcp Source Port | Tcp Destination Port | Udp Source Port | Udp Destination Port |
+| ---- | ------------- | ------------------ | -------- | --------------- | --------------- | -------------------- | --------------- | -------------------- |
+| user_defined_app1 | src_prefix_set1 | dest_prefix_set1 | udp<br>tcp | 25 | src_port_set1 | dest_port_set1 | src_port_set2 | dest_port_set2 |
+| user_defined_app2 | src_prefix_set2 | dest_prefix_set2 | pim<br>icmp<br>tcp | 21<br>7-11 | - | - | - | - |
+
+### Application Profiles
+
+#### Application Profile Name app_profile_1
+
+| Type | Name | Service |
+| ---- | ---- | ------- |
+| application | aim | audio-video |
+| application | aim | chat |
+| application | user_defined_app1 | - |
+| category | best-effort | - |
+| category | category1 | audio-video |
+| transport | http | - |
+| transport | udp | - |
+
+#### Application Profile Name app_profile_2
+
+| Type | Name | Service |
```

Comment:
I think this should be 3 tables instead of this.

Reply:
I had a thought with three tables but looks little scattered having three tables under an application profile.
On suggestion from Guillaume I moved it under a table which looked cleaner to me as well.

Logically this table completely aggregates elements part of an application profile.
I am ok to segregate if it makes more sense to do so.

Reply:
I had a chat with Claus - keep it as it is for now and we will adjust later if needed